app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#6200EE</color>
-    <color name="colorPrimaryDark">#3700B3</color>
-    <color name="colorAccent">#03DAC5</color>
+    <color name="white">#FFFFFFFF</color>
+
+    <color name="black">#FF000000</color>
+
+    <color name="graphite">#FF33373A</color>
```

Comment:
We need to find a solution for color naming before adding more and more colors to the new codebase. Let's agree on something and not use the legacy colors names.

Reply:
Maybe we can name all the colors by function to avoid headaches.
https://www.novatec-gmbh.de/en/blog/name-android-colors-palettes/

Reply:
do we have time for this now? I think that's more or less what Bradley suggested in the planning yesterday. As far as I understand, we decided to first quickly move the codebase, and iteratively change what's needed. @android10 how should we proceed?